test/src/test/kotlin/graphql/nadel/tests/next/fixtures/defer/DeferWithErrorTestSnapshot.kt

```diff
@@ -0,0 +1,161 @@
+// @formatter:off
+package graphql.nadel.tests.next.fixtures.defer
+
+import graphql.nadel.tests.next.ExpectedNadelResult
+import graphql.nadel.tests.next.ExpectedServiceCall
+import graphql.nadel.tests.next.TestSnapshot
+import graphql.nadel.tests.next.listOfJsonStrings
+import kotlin.Suppress
+import kotlin.collections.List
+import kotlin.collections.listOf
+
+/**
+ * This class is generated. Do NOT modify.
+ *
+ * Refer to [graphql.nadel.tests.next.UpdateTestSnapshots
+ */
+@Suppress("unused")
+public class DeferWithErrorTestSnapshot : TestSnapshot() {
```

Comment:
This snapshot doesn't seem to have an accompanying test class

Reply:
That was from an old renamed filed. I've deleted it now